pkg/proxy/iptables/proxier.go

```diff
@@ -1306,40 +1336,28 @@ func (proxier *Proxier) syncProxyRules() {
 
 			if hasEndpoints {
 				args = append(args[:0],
+					"-A", string(kubeNodePortsChain),
```

Comment:
The "before" code added a mark-masq for nodeports to SVC - this drops that rule.  Is this correctly covered by the major  changes above?  It's hard to tell..

Reply:
yes; with the new organization there is a single mark-masq call per service, rather than having a separate rule for each nodeport, external IP, and loadbalancer IP.